pontoon/base/models.py

```diff
@@ -670,6 +676,7 @@ def serialize(self):
             'script': self.script,
             'ms_translator_code': self.ms_translator_code,
             'ms_terminology_code': self.ms_terminology_code,
+            'transvision': json.dumps(self.transvision),
```

Comment:
Is there a reason why do you need to have this boolean as a string?

Reply:
We read this on the client side and without the json trick, we pass `"True"` instead of `true`. Could also output the entire dict as json.

Reply:
That's true, I'm just not sure why you don't serialize the output from `.serialize()` (that could be safer for the rest of the available values).

Reply:
Out of curiosity, where in the client-side is this loaded specifically? Thanks.

Reply:
Here's one place the client loads this: https://github.com/mozilla/pontoon/blob/master/pontoon/base/static/js/translate.js#L3488